sig-robotics/README.md

```diff
@@ -23,18 +23,16 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
 
 The Chairs of the SIG run operations and processes governing the SIG.
 
-*TODO*
+- Fujita Tomoya (@fujitatomoya), Sony
+- Zhengguang Ou (@ugvddm), Huawei
 
 ### Technical Leads
 
 The Technical Leads of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
 
-- Zhengguang Ou (@ugvddm)
 - JoeyHwong (@JoeyHwong-gk)
 
-### Co-Chairs
+> The Chairs and Technical Leads will be elected on an annual basis in November for a one year term. 
```

Comment:
The `term` and `election` will follow the community governance, thanks :)

Reply:
I've made the necessary deletion, thank you for clarifying! 😊